modin/pandas/test/dataframe/conftest.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+from modin.config import StorageFormat
+
+
+def pytest_collection_modifyitems(config, items):
```

Comment:
Doesn't this function have 3 parameters? https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems

Reply:
It also works in this way - https://docs.pytest.org/en/7.1.x/how-to/writing_hook_functions.html. Seems, the config argument could also be removed.